### PR TITLE
sandpit blog post

### PR DESCRIPTION
**Pull request by Mbex**

### CODE REVIEW

File: _posts/2016-09-02-ar-sandpit.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+The idea is this apparatus is accessible to everyone and gives as many people a chance to explore and think about our environment as much as possible.
+
+### Unity
+
+The depth data collected by the Kinect is run through the Unity game engine (behind the indescribably popular game Pokemon GO) controlled by C# scripts to generate a virtual environment that can be projected back onto the sand surface.
+
+The depth data is used by Unity to generate a material that is then skinned onto a surface upon which different textures can be placed.
+
+The texture is controlled on the GPU by shaders (essentially GPU programmes) written in HLSL and are used to change the colour/lighting/display of the projection.
```
Comment (jacobtomlinson) [inline _posts/2016-09-02-ar-sandpit.md]: I think this should be the American spelling.

File: _posts/2016-09-02-ar-sandpit.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+### Whistles and Bells
+
+This project has as many legs as a centipede. We added a slider to control the simulated water level to see what the UK would look like if the ice caps melted or another ice age revealed hidden land from beneath the seas.
+
+The UCDavis AR sand *box* makes full use of the Kinects gesture recognition software allowing the user to wave a hand and drop simulated water onto the projected surface, using the physically real topography of the sand to channel the simulated water.
+
+Unfortunately the libfreenect C# library we used for interacting with the Kinect doesn't have any in built gesture recognition, so we would have to write these ourselves.. which could be something that happens in the future.
+
+![alt text](https://images.informaticslab.co.uk/articles/2016-09-02_ar_sandpit/IMG_1325.JPG "Logo Title Text 1") video of the uk and a slider
```
Comment (jacobtomlinson) [inline _posts/2016-09-02-ar-sandpit.md]: I dont think this is the right image.

**Comment by Mbex** [comment]
I think thats where the video is supposed to go

**Comment by jacobtomlinson** [comment]
Ah! I'll get it on YouTube.

**Comment by jacobtomlinson** [comment]
https://youtu.be/9qcvXsKrJro

**Comment by jacobtomlinson** [comment]
You can embed youtube videos like this `{% youtube <video_id> %}` so in this case it would be `{% youtube 9qcvXsKrJro %}`.

**Pull request merged**